src/pages/index.js

```diff
@@ -41,10 +41,9 @@ const IndexPage = ({data}) => {
         description="You can help decarbonize your state, and it's a lot simpler than you think! Learn how with Decarbonize My State."
       />
       <div className="main-header">
-        <h1 id="main" className='display-4 text-center font-weight-bold'>
-          What does take...</h1>
-        <h1 className='display-4 text-center font-weight-bold'>
-          to decarb your state?
+        <h1 id="main" className="display-4 text-center font-weight-bold">
```

Comment:
We had two `h1` elements, which doesn't make sense. In terms of semantics, that would mean we are saying the page has two titles.